recipes/Media/ASR/seq2seq/train.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/env python3
+
+# ################################
+# Authors modifier: Gaelle Laperriere
```

Comment:
Authors (no need for modifier).

Reply:
Took it from another recipe. Correction done.